.github/workflows/ci.yml

```diff
@@ -10,13 +10,13 @@ jobs:
           - 15
           - 16
         ruby:
-          - "3.1"
```

Comment:
Same comment from here: https://github.com/jcoleman/relation_to_struct/pull/23#discussion_r1857340361

The other versions of Rails still support 3.1 but it was much easier to just remove it completely instead of special-casing Rails 8. I think this should be fine as Ruby 3.1 is EOL in 4 months?